### PR TITLE
Add rate limiting header docs

### PR DESCRIPTION
**Pull request by danihodovic**
## Deployment checklist
- [ ] The rate limiting headers need to be flagged to `on` in production by a change in chef.
- [x] Dani has been able to preview the docs locally which needs to be done to ensure they look OK.
- [x] We still need to decide on whether to send the `x-contentful-ratelimit-reset` header on all request or only 429s for the CMA and CPA.

### CODE REVIEW

File: apiary/cpa.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
```
Comment (neonichu) [inline apiary/cpa.html]: This file shouldn't be checked in.

File: apiary/cma.apib [hunk 2]
```diff
@@ -59,11 +59,21 @@ Contentful uses optimistic locking. When updating an existing resource its curre
 
 ## API rate limits
 
-In the Content Management API, there is a limit of 10 requests per second. The limit exists to prevent unlimited usage and ensure the consumption of resources that a single account can exercise is limited.
+The Content Management API enforces standard rate limits per hour and per second. The limits exist to prevent unlimited usage and ensure the consumption of resources that a single account can exercise is limited. Rate limits are set to 18000 requests per hour and 10 requests per second.
 
-The rate limit is implemented through a standard mechanism of the HTTP protocol (returning response code 429).
+The rate limits are implemented through a standard mechanism of the HTTP protocol (returning response code 429).
 
-:[Common Resource Attribute](_partials/cma/resource-attributes.apib)
```
Comment (neonichu) [inline apiary/cma.apib]: This line shouldn't have been removed, please re-add it after the rate limits section.

**Comment by danihodovic** [comment]
> This file shouldn't be checked in.

Clumsy on my part, fixed in f9cf68d

> This line shouldn't have been removed, please re-add it after the rate limits section.
 
Fixed in b7b0f5b

**Comment by danihodovic** [comment]
Can someone have a :eyes: at this?

**Comment by neonichu** [comment]
LGTM :+1:

File: apiary/cda.apib [hunk 3]
```diff
@@ -21,10 +21,17 @@ As an authorization HTTP request-header field and as an URI query parameter name
 ## API rate limits
 
 Rate limits refer to the amount of requests per unit of time that can be done against our APIs.
-On the Content Delivery API there are no limits enforced on requests that hit the cache (i.e. a cache hit does not count towards rate limit and an unlimited amount of cache hits can be done). Nonetheless, there is still a rate limit of approximately 80 requests per seconds on the origin.
-
+On the Content Delivery API there are no limits enforced on requests that hit the cache (i.e. a cache hit does not count towards rate limit and an unlimited amount of cache hits can be done). Nonetheless, there is still a rate limit of approximately 21600 requests per hour and 80 requests per seconds on the origin.
```
Comment (benben) [inline apiary/cda.apib]: why approximately? we have exact limits set for that.

Comment (danihodovic) [inline apiary/cda.apib]: Because I didn't change the original docs :trollface:

Comment (benben) [inline apiary/cda.apib]: please remove the approximately and use the numbers behind you on the wall :)

Comment (benben) [inline apiary/cda.apib]: maybe we should also add that higher rate limits may apply due to enterprise contracts

**Comment by danihodovic** [comment]
> please remove the approximately and use the numbers behind you on the wall :)
> maybe we should also add that higher rate limits may apply due to enterprise contracts

Fixed in 7b12d14

**Pull request merged**